portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -6,13 +6,44 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+// I added Gson library dependency to pom.xml beforehand, but 
+// I had to still import it here
+import com.google.gson.Gson; 
+
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
 
+    String[] messages = {"I like to learn new skills", 
+        "I love to serve the community",
+        "I believe computer science has the power to better the world",
+        "Badminton is one of my favorite sports",
+        "Singing is something I have been doing from a very young age",
+        "I successfully ran my high school service club for 4 years, helping various communities, along with my team"};
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Sahiti!");
+    // response.setContentType("text/html;");
+    // response.getWriter().println("Hello Sahiti!");
+
+    // Convert the array of messages to JSON
+
+    HelloWorldServlet helloWorld = new HelloWorldServlet();
+    String json = convertToJsonUsingGson(helloWorld);
+
+    // Send the JSON as the response
+    response.setContentType("application/json;"); // correct? or jsonText
```

Comment:
Remove non-final comments from the commit as well. You want the final code to be production quality.